app/controllers/recipes_controller.rb

```diff
@@ -1,69 +1,57 @@
 class RecipesController < ApplicationController
-  before_action :set_recipe, only: %i[show edit update destroy]
+  before_action :set_recipe, only: %i[index show destroy]
```

Comment:
- [ ] Because there is no `set_recipe`, kindly remove this line so your recipe page don't error out